src/range_proof/messages.rs

```diff
@@ -10,6 +10,7 @@ use alloc::vec::Vec;
 use core::iter;
 use curve25519_dalek::ristretto::{CompressedRistretto, RistrettoPoint};
 use curve25519_dalek::scalar::Scalar;
+use curve25519_dalek::traits::VartimeMultiscalarMul;
```

Comment:
why is this additional trait import needed?

Reply:
I actually just moved this value here from line 94 (and removed 1 unused trait). I can move it back again to line 94 if you prefer? This trait is needed to expose the following function on line 128:
```rust
let P_check = RistrettoPoint::vartime_multiscalar_mul(
```

Reply:
That seems fine, thanks for explaining. LGTM